examples/slow_events.cpp

```diff
@@ -0,0 +1,73 @@
+#if !defined(_GLIBCXX_USE_NANOSLEEP)
+  #define _GLIBCXX_USE_NANOSLEEP
+#endif
+
+#include "cpp-terminal/iostream.hpp"
+#include "cpp-terminal/terminal.hpp"
+
+#include <chrono>
+#include <cpp-terminal/event.hpp>
+#include <cpp-terminal/input.hpp>
+#include <cpp-terminal/key.hpp>
+#include <thread>
+
+// Same than event but add 1s to slow down the event processing.
+int main()
```

Comment:
do we also have an equivalent fast_events.cpp?

Reply:
The events.cpp should be the "fast" default one.. the slow is just to stress the fact that key pressed could be merged into one event due to the loop time se had before.